### PR TITLE
[MRG] Fixes

### PR DESCRIPTION
**Pull request by teonlamont**
Minor fixes.

### CODE REVIEW

**Comment by teonlamont** [comment]
addresses https://github.com/mne-tools/mne-python/pull/2234#issuecomment-114761718

File: mne/report.py [hunk 1]
```diff
@@ -551,7 +551,7 @@ def _build_html_slider(slices_range, slides_klass, slider_id):
 footer_template = HTMLTemplate(u"""
 </div></body>
 <div class="footer">
-        &copy; Copyright 2012-2013, MNE Developers.
+        &copy; Copyright 2012-{{current_year}}, MNE Developers.
```
Comment (agramfort) [inline mne/report.py]: you don't need to define current_year elsewhere?

Comment (teonlamont) [inline mne/report.py]: my fault, cherry-picked the first commit, manually did the second and forgot to extract all relevant lines.

**Comment by agramfort** [comment]
Let me know when fixed

**Comment by teonlamont** [comment]
@agramfort fixed. tested it and it works

**Comment by agramfort** [comment]
Thanks

**Pull request merged**